apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,129 @@
+
+Pre-structured data will:
+
+- enable applications to interface with each other without requiring foundation models to perform a 'reconciliation' step, transforming one's outputs to another's inputs
+- provide safety and confidence through verification and validation: even where reconciliation may be conducted by foundation models, having example structured data will enable model-produced translations to be tested
+- provide a cost benefit: data does not need to be structured at the point of need, it already exists
+- provide a time benefit: data is instantly available, again not requiring computation on the fly
+- act as an always-available fallback in the event of foundation model unavailibilty or disconnection
+
+These features of pre-structured data, combined with foundation models, will unlock and mainstream new technologies and best practices.
+
+## Increasing value
+
+With thanks to both foundation models _and_ pre-structured data, we may finally see certain frontiers unlocked. One thing we're betting on at HASH is that **simulation** will finally come of age, as a widely-used tool for everyday decisionmaking.
+
+HASH _began life_ building agent-based modeling software. Back in 2019, we encountered two main problems in convincing folks to adopt predictive, "generative" AI and simulations in their businesses...
+
+### The shape of data
+
+The first big blocker we encountered was the shape of folks' existing data. Of all the folks we engaged with, generally speaking _none_ had data easily available in nice, neat agent-mappable form. The closest we encountered were enteprises who'd deployed software like Palantir's and developed ["dynamic ontologies"](https://hash.ai/glossary/schemas), but even these rarely represented their entire organization's set of information in a semantically meaningful way. In the last couple of years, the term "semantic layer" has become vogue term in some corners of Silicon Valley, and is used to refer to the ideal of a business whose data is enitrely neatly mapped to real-world "entities". This remains an ambition for many, as opposed to a realized state of affairs, and has taken a bit of a backburner to more hypey AI trends of late. Nevertheless, we now believe that **foundation models will help organizations quickly 'type' their data and convert what they have to 'entities'** suitable for use in many contexts, including simulation.
+
+### The cost of coordination
+
+The second major blocker to mainstream development and adoption of simulation is one of coordination. Simulations have historically required handoff between:
+
+1.  data engineers and analysts who make the data available which is required to instantiate 'agents' in a simulation, and hydrate them with 'properties' that describe them accurately;
+2.  programmers who're able to encode subject matter expertise _in code_ that provides 'behavioral logic' to the agents, and environment within which they reside;
```

Comment:
if 'behavioral logic' should be applied to 'environment' I'd drop the comma, otherwise I think something else is needed, e.g.
```suggestion
2.  programmers who're able to encode subject matter expertise _in code_ that provides 'behavioral logic' to the agents, and define the environment within which they reside;
```